src/contexts/events.tsx

```diff
@@ -40,21 +43,35 @@ export interface WalletConnectEvent {
   account_address?: string;
 }
 
-interface OfframpingParameters {
+export interface OfframpingParameters {
   from_asset: string;
   to_asset: string;
   from_amount: string;
   to_amount: string;
 }
 
 export type TransactionEvent = OfframpingParameters & {
-  event: 'transaction_confirmation' | 'kyc_started' | 'kyc_completed' | 'transaction_success' | 'transaction_failure';
+  event:
+    | 'transaction_confirmation'
+    | 'kyc_started'
+    | 'kyc_completed'
+    | 'transaction_success'
+    | 'transaction_failure'
+    | 'compare_quote';
```

Comment:
I think this is not intended. The event where `event` is `compare_quote` is the `CompareQuoteEvent` (as defined further below) and requires additionally the `..._quote` entries.

The only other reason to list it here is to make it callable for `createTransactionEvent`, but we don't do this anyway.